Framework/PythonInterface/plugins/algorithms/MaskBTP.py

```diff
@@ -258,11 +259,17 @@ def _getBankName(self, banknum, validFrom):  # noqa: C901
                     return 'detector1'
                 elif banknum == 2:
                     return 'wing_detector'
+        elif self.instname == "D33":
+            banks = ["back_detector", "front_detector_top", "front_detector_right", "front_detector_bottom",
+                     "front_detector_left"]
+            return banks[banknum]
+        elif self.instname in ["D11", "D11lr", "D22", "D22lr", "D16"]:
```

Comment:
👍  for the low resolution ones! Could you also add tests for those? You might need to specify the IDF by path, since they should have the same name of the instrument. You can get the instrument directory from config.